includes/class-wc-admin-note.php

```diff
@@ -0,0 +1,463 @@
+	 * @return string
+	 */
+	public function get_title( $context = 'view' ) {
+		return $this->get_prop( 'title', $context );
+	}
+
+	/**
+	 * Get note content.
+	 *
+	 * @param  string $context What the value is for. Valid values are 'view' and 'edit'.
+	 * @return string
+	 */
+	public function get_content( $context = 'view' ) {
+		return $this->get_prop( 'content', $context );
+	}
+
+	/**
+	 * Get note icon (Gridicon).
+	 *
+	 * @param  string $context What the value is for. Valid values are 'view' and 'edit'.
+	 * @return string
+	 */
+	public function get_icon( $context = 'view' ) {
+		return $this->get_prop( 'icon', $context );
+	}
+
+	/**
+	 * Get note content data (i.e. values that would be needed for re-localization)
+	 *
+	 * @param  string $context What the value is for. Valid values are 'view' and 'edit'.
+	 * @return array
+	 */
+	public function get_content_data( $context = 'view' ) {
+		return $this->get_prop( 'content_data', $context );
+	}
+
+	/**
+	 * Get note status.
+	 *
+	 * @param  string $context What the value is for. Valid values are 'view' and 'edit'.
+	 * @return string
+	 */
+	public function get_status( $context = 'view' ) {
+		return $this->get_prop( 'status', $context );
+	}
+
+	/**
+	 * Get note source.
+	 *
+	 * @param  string $context What the value is for. Valid values are 'view' and 'edit'.
+	 * @return string
+	 */
+	public function get_source( $context = 'view' ) {
+		return $this->get_prop( 'source', $context );
+	}
+
+	/**
+	 * Get date note was created.
+	 *
+	 * @param  string $context What the value is for. Valid values are 'view' and 'edit'.
+	 * @return WC_DateTime|NULL object if the date is set or null if there is no date.
+	 */
+	public function get_date_created( $context = 'view' ) {
+		return $this->get_prop( 'date_created', $context );
+	}
+
+	/**
+	 * Get date on which user should be reminded of the note (if any).
+	 *
+	 * @param  string $context What the value is for. Valid values are 'view' and 'edit'.
+	 * @return WC_DateTime|NULL object if the date is set or null if there is no date.
+	 */
+	public function get_date_reminder( $context = 'view' ) {
+		return $this->get_prop( 'date_reminder', $context );
+	}
+
+	/**
+	 * Get actions on the note (if any).
+	 *
+	 * @param  string $context What the value is for. Valid values are 'view' and 'edit'.
+	 * @return array
+	 */
+	public function get_actions( $context = 'view' ) {
+		return $this->get_prop( 'actions', $context );
+	}
+
+	/*
+	|--------------------------------------------------------------------------
+	| Setters
+	|--------------------------------------------------------------------------
+	|
+	| Methods for setting note data. These should not update anything in the
+	| database itself and should only change what is stored in the class
+	| object.
+	|
+	*/
+
+	/**
+	 * Set note name.
+	 *
+	 * @param string $name Note name.
+	 */
+	public function set_name( $name ) {
+		// Don't allow empty names
```

Comment:
Missing period.

Reply:
Fixed in https://github.com/woocommerce/wc-admin/pull/464/commits/4433670cb7eb3ecf2b4d05ed34b83b6ab4e6dea8